### ISSUE TITLE
iOS version?

### ISSUE THREAD

**Issue by NathanHazout** [aws-samples/amazon-bedrock-client-for-mac#29]
Is there an equivalent project for iOS?

**Comment by didhd** [comment]
@NathanHazout I have been considering a version for iOS, but I haven't actually created the project yet. It's likely that AWS SSO would be mandatory for iOS. I plan to first implement AWS SSO login for the Mac client and then look into developing the iOS version.

**Comment by zhu-xiaowei** [comment]
@NathanHazout You can try this project, which supports Android in addition to iPhone
https://github.com/aws-samples/swift-chat

**Comment by didhd** [comment]
Yes. The separate [AI Assistant project](https://github.com/aws-samples/sample-mobile-ai-assistant), previously linked here as `aws-samples/swift-chat`, supports iOS and Android. The old repository URL now redirects to that project, and its current README includes iOS build instructions.

Closing this as an answered question. This repository contains the native macOS client; the mobile implementation is maintained in the linked project.
